### PR TITLE
Remove old awards

### PR DESCRIPTION
**Pull request by g4s8**
#386 - remove awards which are older than 90 days

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#776](https://github.com/zerocracy/farm/issues/776) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#776](https://github.com/zerocracy/farm/issues/776) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/672054509). The budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4). Please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

**Comment by carlosmiranda** [comment]
@yegor256 good to merge

File: src/main/java/com/zerocracy/pmo/Awards.java [hunk 1]
```diff
@@ -82,6 +107,26 @@ public Awards bootstrap() throws IOException {
      * @throws IOException If fails
      */
     public void add(final int points, final String job, final String reason)
+        throws IOException {
+        this.add(
+            points,
+            job,
+            reason,
+            Date.from(Instant.ofEpochMilli(System.currentTimeMillis()))
```
Comment (yegor256) [inline src/main/java/com/zerocracy/pmo/Awards.java]: @g4s8 may I ask, why not just `new Date()`? I do love Java 8+, but wtf?)

File: src/main/resources/com/zerocracy/stk/pmo/awards/refresh_awards.groovy [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import com.zerocracy.farm.Assume
+import com.zerocracy.pm.ClaimIn
+import com.zerocracy.pmo.Awards
+import com.zerocracy.pmo.People
+
+def exec(Project pmo, XML xml) {
+  new Assume(pmo, xml).isPmo()
+  new Assume(pmo, xml).type('Ping daily')
+  ClaimIn claim = new ClaimIn(xml)
+  Date outdated = claim.created() - 90
```
Comment (yegor256) [inline src/main/resources/com/zerocracy/stk/pmo/awards/refresh_awards.groovy]: @g4s8 let's use `new Policy().get("18.days", 90)`

**Comment by yegor256** [comment]
@g4s8 @carlosmiranda  see 8da68d13

File: src/main/java/com/zerocracy/pmo/Awards.java (hunk 1 above)
Comment (g4s8) [reply src/main/java/com/zerocracy/pmo/Awards.java]: @yegor256 sorry, I was tangled between old and new date-time API :)

**Comment by g4s8** [comment]
@yegor256 fixed, please see new changes

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#776](https://github.com/zerocracy/farm/issues/776) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)

**Comment by ypshenychka** [comment]
@carlosmiranda According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

No issues were found during code review. 
Please confirm that you'll try to find at least three problems while future reviews.

**Comment by carlosmiranda** [comment]
@ypshenychka confirmed

**Comment by ypshenychka** [comment]
@carlosmiranda thanks

**Comment by ypshenychka** [comment]
@0crat quality bad

**Comment by 0crat** [comment]
Quality is low, no payment, see [§31](http://www.zerocracy.com/policy.html#31)

**Comment by 0crat** [comment]
Quality review completed: +8 points just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**